scout/server/blueprints/panels/templates/panels/panel_pdf_simple.html

```diff
@@ -18,25 +18,28 @@ <h4>Scout - Gene panel report</h4> - created on:&nbsp;<strong>{{report_created_a
   <div class="card-body">
     <table class="table table-sm">
       <tr>
-        <td>Updated:<span class="badge bg-secondary rounded-pill text-white">{{ panel.date.strftime('%Y-%m-%d') }}</span></td>
```

Comment:
The fix is the style of these spans, which is no longer recognized. I've devided to remove the spans, so it's easier to maintain